definitions/npm/zxcvbn_v4.x.x/test_zxcvbn.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import zxcvbn from 'zxcvbn';
+import type { Result, Score, AttackTime, Feedback } from 'zxcvbn';
+import { describe, it } from 'flow-typed-test';
+
+describe('password', () => {
+  const testA: Result = zxcvbn('123456');
+  const guesses: number = testA.guesses;
+  const guessesLog10: number = testA.guesses_log10;
+  const score: Score = testA.score;
+  const calcTime: number = testA.calc_time;
+  const crackTimeSeconds: AttackTime = testA.crack_times_seconds;
+  const onlineThrottling100PerHour =
+    crackTimeSeconds.online_throttling_100_per_hour;
+  const feedback: Feedback = testA.feedback;
+  const warning: string = feedback.warning;
+  const suggestions: string[] = feedback.suggestions;
+});
+
+describe('user input', () => {
+  const test2: Result = zxcvbn('123456', ['abc']);
+});
```

Comment:
Would be nice to see also failing test cases, see https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#4-write-a-test-file-whose-name-starts-with-test_-in-the-package-version-directory